crates/viewer/re_viewport_blueprint/src/viewport_blueprint.rs

```diff
@@ -83,7 +83,7 @@ pub struct ViewportBlueprint {
 
 impl ViewportBlueprint {
     /// Attempt to load a [`ViewBlueprint`] from the blueprint store.
-    pub fn try_from_db(blueprint_db: &re_entity_db::EntityDb, query: &LatestAtQuery) -> Self {
+    pub fn from_db(blueprint_db: &re_entity_db::EntityDb, query: &LatestAtQuery) -> Self {
```

Comment:
nice catch, but there's no attempting here, comment needs update as well
(gh doesn't let me do suggestion here for some reason)